src/app/policies/views/PolicyListView.vue

```diff
@@ -370,10 +366,10 @@ async function getEntity(selectedEntity: { mesh: string, path: string, name: str
       const selected = ['type', 'name', 'mesh']
 
       entity.value = getSome(item, selected)
-      QueryParameter.set('policy', entity.value.name)
+      QueryParameter.set('policy', selectedEntity.name)
       rawEntity.value = stripTimes(item)
     } else {
-      entity.value = {}
+      entity.value = null
```

Comment:
Could you let me know what this was for? Is it a bugfix or something else?

Reply:
It’s just adopting the style of indicating "entity is not currently available" that we use in all the other 5 instances of list view component. It’s not a functional difference in this component as all template conditionals are currently also covered by checks against `isEmpty` (which technically is redundant information) and the change from `:selected-entity-name="entity.name"` to `:selected-entity-name="entity?.name"` is also not different because both will evaluate to `undefined`.